src/components/ui/NoResults/NoResults.tsx

```diff
@@ -10,8 +10,10 @@ export const NoResults: FC = () => {
     <article className={style.container}>
       <h3>No Results Found</h3>
       <p>
-        Could not find a match for
-        <span className={style.search_text}>{` '${searchValue}'`}</span>
+        Could not find a match
+        {searchValue ? (
+          <span className={style.search_text}>{` for '${searchValue}'`}</span>
+        ) : null}
```

Comment:
Try to simplify this condition, using **&&**

Reply:
Substituted ternary with &&